<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellSecond</class>
 <widget class="QDialog" name="SellSecond">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>退货</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>154</y>
     <width>560</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>商品编码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品库存量</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="waterTxt">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>21</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>142</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>21</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退货流水号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>21</y>
     <width>93</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>    退货时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>21</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>354</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>总金额：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sumPriceLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>总积分：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sumScoreLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="returnBtn">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>确认退货</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>59</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>会员卡号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="VIPNumTxt">
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>   已有积分：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="scoreGotTxt">
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>查看</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>561</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品编码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="barcodeTxt">
      <property name="maxLength">
       <number>13</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  退货数量：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="returnTxt"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
